workspaces/servicenow/plugins/servicenow/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "@backstage-community/plugin-servicenow",
+  "version": "0.1.0",
+  "license": "Apache-2.0",
+  "private": true,
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/community-plugins",
+    "directory": "workspaces/servicenow/plugins/servicenow"
+  },
+  "backstage": {
+    "role": "frontend-plugin",
+    "pluginId": "servicenow",
+    "pluginPackages": [
+      "@backstage-community/plugin-servicenow",
+      "@backstage-community/plugin-servicenow-backend",
+      "@backstage-community/plugin-servicenow-common"
+    ]
+  },
+  "sideEffects": false,
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack"
+  },
+  "dependencies": {
+    "@backstage-community/plugin-servicenow-common": "workspace:^",
+    "@backstage/catalog-model": "^1.7.4",
+    "@backstage/core-components": "^0.17.2",
+    "@backstage/core-plugin-api": "^1.10.7",
+    "@backstage/plugin-catalog-react": "^1.18.0",
+    "@backstage/theme": "^0.6.6",
+    "@mui/icons-material": "^5.15.17",
+    "@mui/lab": "^5.0.0-alpha.153",
+    "@mui/material": "^5.17.1",
+    "@mui/styles": "5.16.14",
+    "react-dom": "^16.13.1 || ^17.0.0 || ^18.0.0",
+    "react-router-dom": "^6.0.0",
+    "react-use": "^17.2.4"
+  },
+  "peerDependencies": {
+    "react": "^16.13.1 || ^17.0.0 || ^18.0.0",
+    "react-dom": "^16.13.1 || ^17.0.0 || ^18.0.0",
+    "react-router-dom": "^6.0.0"
+  },
+  "devDependencies": {
+    "@backstage/cli": "^0.32.1",
+    "@backstage/core-app-api": "^1.17.0",
+    "@backstage/dev-utils": "^1.1.10",
+    "@backstage/test-utils": "^1.7.8",
+    "@material-ui/core": "^4.12.4",
+    "@testing-library/jest-dom": "^6.0.0",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/user-event": "^14.0.0",
+    "msw": "^1.0.0",
+    "react": "^16.13.1 || ^17.0.0 || ^18.0.0",
+    "rxjs": "^7.8.2"
+  },
+  "files": [
+    "dist"
+  ],
+  "scalprum": {
+    "name": "backstage-community.plugin-servicenow",
+    "exposedModules": {
+      "PluginRoot": "./src/index.ts",
+      "isMyProfile": "./src/utils/isMyProfile.ts"
+    }
+  },
```

Comment:
What is this? Haven't seen this before 😅 does it need to be here?

Reply:
I believe you can export the isMyProfile function also in `workspaces/servicenow/plugins/servicenow/src/index.ts`

```tsx
export { servicenowPlugin, ServicenowPage } from './plugin'; // I personally like to export everything from ./plugin
export { isMyProfile } from './utils/isMyProfile';
```

Reply:
This was something I tried which didn't work. I already found the correct way to use the condition module in our dynamic plugin system. I have had this removed 👍